### ISSUE TITLE
Style property 'width' is not supported by native animated module

### ISSUE THREAD

**Issue by anmolguptachicmic** [mxck/react-native-material-menu#88]
**Issue Description**
When I click on the menu button it gives me an error **Style property 'width' is not supported by native animated module**
This was working fine in 1.0.0 but it is broken in version 1.1.1

**Expected Results**
Should show the menu on click

**Additional Information**
React Native version:0.61
Platform(s) (iOS, Android, or both?):both
Operating System (macOS, Linux, or Windows?): macOS

**Steps to Reproduce / Code Snippets**
```
 <Menu
          ref={this.setMenuRef}
          button={<Text onPress={this.showMenu}>Show menu</Text>}
>
          <MenuItem onPress={this.hideMenu}>Edit Comment</MenuItem>
          <MenuItem onPress={this.hideMenu}>Delete Comment</MenuItem>
</Menu>
```
![Simulator Screen Shot - iPhone 8 - 2020-04-02 at 15 58 31](https://user-images.githubusercontent.com/58215818/78239148-005e1300-74fb-11ea-9eab-2ff4eaebd934.png)

**Comment by joshuadiezmo** [comment]
any fixed for this?

**Comment by joshuadiezmo** [comment]
The problem is they add `useNativeDriver: true,` on Menu.js file

@anmolguptachicmic you can just comment this lines

https://github.com/mxck/react-native-material-menu/commit/4e4e7673ac96882a46eeba841f56d547ef07fcaf#diff-e359c93bc52c848fbef7e3bdc25527dbR68

https://github.com/mxck/react-native-material-menu/commit/4e4e7673ac96882a46eeba841f56d547ef07fcaf#diff-e359c93bc52c848fbef7e3bdc25527dbR74

https://github.com/mxck/react-native-material-menu/commit/4e4e7673ac96882a46eeba841f56d547ef07fcaf#diff-e359c93bc52c848fbef7e3bdc25527dbR104

**Comment by yohix** [comment]
This is happening for me too!

**Comment by anmolguptachicmic** [comment]
> The problem is they add `useNativeDriver: true,` on Menu.js file
> 
> @anmolguptachicmic you can just comment this lines
> 
> [4e4e767#diff-e359c93bc52c848fbef7e3bdc25527dbR68](https://github.com/mxck/react-native-material-menu/commit/4e4e7673ac96882a46eeba841f56d547ef07fcaf#diff-e359c93bc52c848fbef7e3bdc25527dbR68)
> 
> [4e4e767#diff-e359c93bc52c848fbef7e3bdc25527dbR74](https://github.com/mxck/react-native-material-menu/commit/4e4e7673ac96882a46eeba841f56d547ef07fcaf#diff-e359c93bc52c848fbef7e3bdc25527dbR74)
> 
> [4e4e767#diff-e359c93bc52c848fbef7e3bdc25527dbR104](https://github.com/mxck/react-native-material-menu/commit/4e4e7673ac96882a46eeba841f56d547ef07fcaf#diff-e359c93bc52c848fbef7e3bdc25527dbR104)

Thanks, @joshuadiezmo it worked.

**Comment by mxck** [comment]
Fixed in 1.1.2! Sorry for 1.1.1, I revoke this version from npm.

**Comment by yohix** [comment]
Thanks a lot, @mxck :heart:
